### PR TITLE
bug fix: trimmed_mean

### PR DESCRIPTION
**Pull request by DataAI**
My attempt to fix the issue #26 , where trimmed_mean estimate is not accurate. I've also added some tests.

### CODE REVIEW

File: tests/test_tdigest.py [hunk 1]
```diff
@@ -174,6 +176,21 @@ def test_ints(self):
...
+    def test_trimmed_mean(self, percentile_range, data_size):
+        p1 = percentile_range[0]
+        p2 = percentile_range[1]
+
+        t = TDigest()
+        x = random.random(size=data_size)
+        t.batch_update(x)
+
+        tm_actual = t.trimmed_mean(p1, p2)
+        tm_expected = x[bitwise_and(x >= percentile(x, p1), x <= percentile(x, p2))].mean()
```
Comment (CamDavidsonPilon) [inline tests/test_tdigest.py]: clever, I've never used `bitwise_and` before

File: tests/test_tdigest.py [hunk 2]
```diff
@@ -174,6 +176,21 @@ def test_ints(self):
         assert t.percentile(50) == 2
         assert sum([c.count for c in t.C.values()]) == len(x)
 
+    @pytest.mark.parametrize("percentile_range", [[0, 7], [27, 47], [39, 66], [81, 99], [77, 100], [0,100]])
+    @pytest.mark.parametrize("data_size", [100, 1000, 5000])
```
Comment (CamDavidsonPilon) [inline tests/test_tdigest.py]: TIL `parameterize` in pytest

**Comment by CamDavidsonPilon** [comment]
Awesome, great tests too. I'll merge this, then update the version and send to pypi!

**Comment by CamDavidsonPilon** [comment]
You don't have much on your profile, but are you looking for a job right now? =)

**Comment by DataAI** [comment]
It's a new profile. :-) I rarely contribute to open source projects because of my current job.

**Pull request merged**